scripts/test-prod.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+port=4873
+original_registry=`npm get registry`
+registry="http://localhost:$port"
+output="output.out"
+commit_to_revert="HEAD"
+test_app='rsp-cra-18'
+
+# Generate dists for the packages
+yarn parcel build packages/@adobe/react-spectrum/ packages/@react-{spectrum,aria,stately}/*/ packages/@internationalized/*/ --no-optimize --log-level error
+
+# Start verdaccio and send it to the background
+yarn verdaccio --listen $port &>${output}&
```

Comment:
This is very close, but you'll want to add `verdaccio` to the overall project's dev dependencies, otherwise this line won't run properly